.github/workflows/main.yml

```diff
@@ -16,11 +15,108 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v3
 
+      - name: Read Review Guidelines
+        id: read-guidelines
+        run: |
+          REVIEW_GUIDELINES=$(cat REVIEW_GUIDE.md)
+          echo "::set-output name=review_guidelines::$REVIEW_GUIDELINES"
+
       - name: AI Code Review Action
-        uses: freeedcom/ai-codereviewer@v2.6.0
+        # uses: freeedcom/ai-codereviewer@v2.6.0
+        uses: piyushkukreja/ai-codereviewer@main
```

Comment:
It seems like you're switching the AI Code Review Action from `freeedcom/ai-codereviewer@v2.6.0` to `piyushkukreja/ai-codereviewer@main`. Please ensure that the new repository and branch have the necessary updates and are stable for use. If possible, use a specific version tag instead of `main` to avoid unexpected changes in the future.